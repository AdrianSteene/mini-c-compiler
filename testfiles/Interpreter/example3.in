int g(int x) { return x; }

int h(int x) { return x*2; }

int i(int x) { return x*3; }

int f(int x, int y, int z) {
  print(g(x));
  print(h(y));
  print(i(z));
  return 0;
}

int main() {
  f(12321, 32123, 11111);
  return 0;
}