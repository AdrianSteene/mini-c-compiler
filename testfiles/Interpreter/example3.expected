12321
64246
33333
